types/pdf-image/index.d.ts

```diff
@@ -0,0 +1,300 @@
+// Type definitions for pdf-image 2.0
+// Project: https://github.com/mooz/node-pdf-image#readme
+// Definitions by: NickLatkovich <https://github.com/nicklatkovich>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.4
```

Comment:
Does this really need 3.4 to work? 2.8 is the earliest supported version on DT and it *looks* like these types might work fine with it.